src/vegas_mode/coordinator.py

```
@@ -0,0 +1,654 @@
+        plugin_id = next_segment.plugin_id
+        plugin = self.plugin_manager.get_plugin(plugin_id)
+
+        if not plugin:
+            return None
+
+        # Check if this plugin is configured for STATIC mode
+        try:
+            display_mode = plugin.get_vegas_display_mode()
+            if display_mode == VegasDisplayMode.STATIC:
+                return plugin
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)